app/src/main/java/org/oppia/android/app/help/thirdparty/LicenseListActivity.kt

```diff
@@ -0,0 +1,45 @@
+package org.oppia.android.app.help.thirdparty
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The activity that will show list of licenses corresponding to a third-party dependency. */
+class LicenseListActivity : InjectableAppCompatActivity(), RouteToLicenseTextListener {
+
+  @Inject
+  lateinit var licenseListActivityPresenter: LicenseListActivityPresenter
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    activityComponent.inject(this)
+    val dependencyIndex = intent.getIntExtra(THIRD_PARTY_DEPENDENCY_INDEX, 0)
+    licenseListActivityPresenter.handleOnCreate(dependencyIndex)
+  }
+
+  companion object {
+    private const val THIRD_PARTY_DEPENDENCY_INDEX = "ThirdPartyListActivity.dependencyIndex"
```

Comment:
This should be 
`     private const val THIRD_PARTY_DEPENDENCY_INDEX = "ThirdPartyListActivity.dependency_index"`

Reply:
Done.